committee-code-of-conduct/bootstrapping-process.md

```diff
@@ -0,0 +1,36 @@
+# Bootstrapping Document for Operator Framework Code of Conduct Committee
+
+This document (created by the Operator Framework community chairs) outlines what the Code of Conduct Committee (CoCC) is responsible for, how to create it, and the rough processes for them to get started.
```

Comment:
this sentence is difficult to parse... `is response for, how to create it, and`
The object initially is "this document"... so it is unclear what "it" means in "create it".

Reply:
I think "how to create it" means "to create the committee from scratch". I agree, will smooth this out.

Reply:
How about:  This document, created by the Operator Framework community chairs, outlines the creation process, mission, and scope of the Code of Conduct Commitee (COCC).